tests/frontend/compilers/test_gaussianunitary.py

```diff
@@ -46,7 +46,8 @@ def random_params(size, sq_bound, disp_bound):
 
 @pytest.mark.parametrize("depth", [1, 3, 6])
 @pytest.mark.parametrize("width", [5, 10, 15])
-def test_gaussian_program(depth, width):
+@pytest.mark.parametrize("compiler", ["gaussian_unitary", "gaussian_merge"])
+def test_gaussian_program(depth, width, compiler):

```

Comment:
@federico0112 : you should do this in each every of the tests in this file. I did it locally and all but `test_non_primitive_gates` pass. You will need to fix that one. I think is because the `MZgate` is included in the wrong place.